### PR TITLE
doc_fragments: Clean up parameter types (other)

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This PR includes:
- Parameter types added
- Copyright format fixes
- Short license statement
- Description fixes (only for a few files, then I stopped :-))

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
module_utils

### CODE REVIEW

**Comment by mattclay** [comment]
@dagwieers I haven't figured out why, but the first commit in this PR is causing the FreeBSD test failures. Running the tests with that commit will consistently fail, while running with the prior commit will consistently pass.

**Comment by mattclay** [comment]
I've also verified that no other PRs or merge runs are experiencing this issue. It is isolated to just this PR.

**Comment by dagwieers** [comment]
@mattclay Thanks for the feedback. I'll investigate.

**Comment by dagwieers** [comment]
I don't know what to do here honestly. Except for opening a PR for each file individually ?

**Comment by mattclay** [comment]
@dagwieers It will be easier to run the tests locally to track this down.

```shell
git checkout 46e4d13ccf7762fa162535772a5b812d1ba3a0b1
ansible-test integration become --remote freebsd/11.1 -v
```

Then undo half of the remaining changes and repeat the test run until problem goes away. Then keep only the changes you just removed and repeat the process. You should only need about 6 test runs to find the problem file.

File: lib/ansible/module_utils/k8s/scale.py [hunk 1]
```diff
@@ -37,7 +22,7 @@ class KubernetesException(Exception):
 SCALE_ARG_SPEC = {
     'replicas': {'type': 'int', 'required': True},
     'current_replicas': {'type': 'int'},
-    'resource_version': {},
+    'resource_version': {'typ': 'str'},
```
Comment (fabianvf) [inline lib/ansible/module_utils/k8s/scale.py]: Should this be `type`?

Comment (dagwieers) [reply lib/ansible/module_utils/k8s/scale.py]: Yup, good catch. Fixed. Thanks!

**Review by samccann — APPROVED** [review]

**Pull request merged**